src/main/java/task.java

```diff
@@ -0,0 +1,59 @@
+public class task {
+    protected String name;
+    private Integer number;
+    private boolean isDone;
+    private String taskType;
+
+    public task(String name, Integer number, boolean isDone, String taskType) {
+        setName(name);
+        setNumber(number);
+        setDone(isDone);
+        setTaskType(taskType);
+    }
+
+    public void setTaskType(String taskType) {
+        this.taskType = taskType;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setNumber(Integer number) {
+        this.number = number;
+    }
+
+    public void setDone(boolean isDone) {
+        this.isDone = isDone;
+    }
+
+    public String getTaskType() {
+        return taskType;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Integer getNumber() {
+        return number;
+    }
+
+    public String getIsDone() {
+        if (isDone) {
+            return "[✓]";
+        } else {
+            return "[✗]";
+        }
+    }
+
+    public String date() {
+        return "";
+    }
```

Comment:
Strange that you default to returning an empty string for dates
Perhaps you should refactor the empty string into a constant so that it is explicit that the empty string is the default